.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -24,17 +24,34 @@ body:
       label: Which component(s) of AVD impacted
       multiple: true
       options:
```

Comment:
IMO the old sorting was better, since the most used is first.

Reply:
resolved